third_party/terraform/tests/resource_container_cluster_test.go.erb

```diff
@@ -1344,6 +1344,15 @@ func TestAccContainerCluster_withIPAllocationPolicy_createSubnetwork(t *testing.
 				ImportState:		 true,
 				ImportStateVerify:	 true,
 			},
+			{
+				Config: testAccContainerCluster_withIPAllocationPolicy_createSubnetworkUpdated(cluster),
```

Comment:
From what I can tell, this test is to confirm that  you can move things out of the old block and set it by using the `subnetwork` field. Would a `PlanOnly` test make more sense here (to check a lack of diffs), or are we expecting the change to actually perform an update?  Also if I'm way off the mark, a comment explaining the purpose of the test would be appreciated.

Reply:
You got it right- I'd forgotten that `PlanOnly` was a thing.